src/main/java/nimbus/ui/Ui.java

```diff
@@ -0,0 +1,78 @@
+package nimbus.ui;
+
+import nimbus.exception.WrongDateTimeFormatException;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Class that contains some of the UI elements needed for the project
+ */
+
+public class Ui {
+    // Came up with the name and used an online ASCII art generator
+    // Generator: https://patorjk.com/software/taag/#p=display&h=0&v=0&f=Ghost&t=nimbus
+    private static String logo = "\n" +
+            "     .-') _             _   .-')    .-. .-')                  .-')    \n" +
+            "    ( OO ) )           ( '.( OO )_  \\  ( OO )                ( OO ).  \n" +
+            ",--./ ,--,'    ,-.-')   ,--.   ,--.) ;-----.\\   ,--. ,--.   (_)---\\_) \n" +
+            "|   \\ |  |\\    |  |OO)  |   `.'   |  | .-.  |   |  | |  |   /    _ |  \n" +
+            "|    \\|  | )   |  |  \\  |         |  | '-' /_)  |  | | .-') \\  :` `.  \n" +
+            "|  .     |/    |  |(_/  |  |'.'|  |  | .-. `.   |  |_|( OO ) '..`''.) \n" +
+            "|  |\\    |    ,|  |_.'  |  |   |  |  | |  \\  |  |  | | `-' /.-._)   \\ \n" +
+            "|  | \\   |   (_|  |     |  |   |  |  | '--'  / ('  '-'(_.-' \\       / \n" +
+            "`--'  `--'     `--'     `--'   `--'  `------'    `-----'     `-----'  \n";
```

Comment:
I like the logo!

Reply:
Thank you! You can get the logo from the website I commented too